src/Compilers/Core/Portable/PEWriter/ContentId.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using System.Diagnostics;
+
+namespace Microsoft.Cci
+{
+    internal struct ContentId
+    {
+        public readonly byte[] Guid;
```

Comment:
It is much easier to (1) copy the data into a stream, and (2) make a Guid, from this format.